### PR TITLE
WIP Add option to download videos and include in export tarball

### PR DESCRIPTION
**Pull request by thallada**

### CODE REVIEW

File: edxval/api.py [hunk 1]
```diff
@@ -900,9 +918,12 @@ def import_from_xml(xml, edx_video_id, course_id=None):
                 profile_name
             )
             continue
+        url = encoded_video_el.get('url')
+        if url.startswith(EXPORTED_VIDEO_PREFIX):
+            url = url.replace(EXPORTED_VIDEO_PREFIX, IMPORTED_VIDEO_PREFIX)
```
Comment (pwnage101) [inline edxval/api.py]: The problem with this is that `olx://video/<video-id>/<video>` will get stored in the DB as <code>local-video://**video/**\<video-id>/\<video></code>.  This is extraneous information, and complicates the video xblock code which currently has to clean up that prefix:

https://github.com/edx/edx-platform/blob/e5270f790/common/lib/xmodule/xmodule/video_module/video_module.py#L1140

**Comment by pwnage101** [comment]
We should probably move the code which imports the video files:

https://github.com/edx/edx-platform/blob/e5270f7909/cms/djangoapps/contentstore/tasks.py#L506-L525

into this repository since the most of the other video import/export logic is already here.

**Comment by DawoudSheraz** [comment]
@pwnage101 / @thallada Hi. I see this PR has been opened for quite some time with no action or updates. Are there any objections to closing this PR? Thank you

**Comment by DawoudSheraz** [comment]
Closing this PR. It can be re-opened in the future if necessary.

**Pull request closed without merging**